packages/auth/src/model/public_types.ts

```diff
@@ -280,6 +280,12 @@ export interface Auth {
     callback: (user: User | null) => void | Promise<void>,
     onAbort?: () => void
   ): Unsubscribe;
+
+  onFirebaseTokenChanged(
```

Comment:
Related: this method was not part of the API review. I'm concerned that `onFirebaseTokenChanged` (and the concept of a `FirebaseToken`) is very generic sounding. I'm worried it'll confuse users.

Reply:
We should go through another API review process surfacing this. "firebaseToken" is a really generic name -- "Firebase" is a huge suite of products that has many different types of tokens.